frameworks/youth-risk-assessment/questions/health-affecting-transport.yml

```diff
@@ -0,0 +1,78 @@
+type: checkbox
+question: Does this person’s health affect transport?
+description: Health affecting transport
+hint: Select all that apply
+prefill: true
+options:
+  -
+    label: Special diet or allergies
+    value: Special diet or allergies
+    followup_comment:
+      label: Give details
+      validations:
+        -
+          type: required
+          message: Enter details
```

Comment:
Might be worth expanding on these error messages too to prevent `Enter details` x5